src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -56,9 +56,9 @@ export interface IFacetSliderOptions extends ISliderOptions, IResponsiveComponen
  * > [`getSteps`]{@link FacetSlider.options.getSteps} and [`valueCaption`]{@link FacetSlider.options.valueCaption})
  * > cannot be configured as `data-` attributes in the markup. If you wish to configure those options, you must either
  * > do so in the [`init`]{@link init} call of your search interface (see
- * > [Components - Passing Component Options in the init Call](https://developers.coveo.com/x/PoGfAQ#Components-PassingComponentOptionsintheinitCall)),
+ * > [Components - Passing Component Options in the init Call](https://docs.coveo.com/en/346/#passing-component-options-in-the-init-call)),
```

Comment:
You can remove "Components - "

Same comment for other similar occurrences in this PR.